### DISCUSSION TITLE
How can we use Polymorphic relation MorphToMany with attachments on FileUpload?

### DISCUSSION THREAD

**Discussion by zeeshantariq08** [filamentphp/filament#9688] [Help]
Attachments( uploads) for multiple models - MorphToMany

**Comment by dr-codswallop** [comment]
I'm also stumped with this at the moment

**Reply by zeeshantariq08** [reply]
I am working on it , hope i will finish it

**Reply by dr-codswallop** [reply]
Great. Please share your solution if you work it out 🙏 good luck
